lib/index.js

```diff
@@ -59,14 +67,15 @@ addExerciseButton.on('click', function (event) {
   var calories = $('input[name=calories]').val();
   var exercise = new Exercise(name, calories);
   exercise.store();
-  addToTable(name, calories);
+  addExerciseToTable(name, calories);
   $('input[name=name]').val('');
   $('input[name=calories]').val('');
 });
 
-function addToTable(name, calories) {
-  var row = '<tr><td>' +name+ '</td><td>' +calories+ '</td><td><a href="#" class="delete-exercise material-icons">delete</a></td></tr>'
+function addExerciseToTable(name, calories) {
+  var row = '<tr id="editable" class="new-row" ><td contenteditable="true">' +name+ '</td><td contenteditable="true">' +calories+ '</td><td><a href="#" class="delete-exercise material-icons">delete</a></td></tr>'
```

Comment:
@MDes41 if possible I think the id bit in here should be changed. ids are meant to be unique and the way that this is setup each row will have the same id. Perhaps it could be a class name instead?